ontology/yaml/resources/METERS/entity_types/ABSTRACT.yaml

```diff
@@ -22,6 +22,8 @@ PWM:
   opt_uses:
   - apparent_power_sensor
   - reactive_power_sensor
+  - apparentpower_sensor
```

Comment:
why did you add these when there are apparent and reactive power fields defined on the lines above?

Reply:
Sorry, I wanted to see them side by side and is easier to remove them later when you determine which of them are valid. As it is, I will now use apparent_power...and reactive_power... thanks sir.